### PR TITLE
Slash command for ,profile

### PR DESCRIPTION
**Pull request by taahamahdi**

### CODE REVIEW

File: src/commands/game_commands/profile.ts [hunk 1]
```diff
@@ -318,6 +319,34 @@ export default class ProfileCommand implements BaseCommand {
...
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: "user_mention",
+                    description: "Mention the user whose profile to view",
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .MENTIONABLE,
+                    required: false,
+                    channel_types: undefined as never,
```
Comment (taahamahdi) [inline src/commands/game_commands/profile.ts:337]: I can't get the compiler to stop complaining without this line

File: src/commands/game_commands/profile.ts [hunk 2]
```diff
@@ -318,6 +319,34 @@ export default class ProfileCommand implements BaseCommand {
...
+            name: "profile",
+            description: LocalizationManager.localizer.translateByLocale(
+                LocaleType.EN,
+                "command.profile.help.description"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: "user_mention",
+                    description: "Mention the user whose profile to view",
```
Comment (Brainicism) [inline src/commands/game_commands/profile.ts:333]: Translations missing

File: src/commands/game_commands/profile.ts [hunk 3]
```diff
@@ -318,6 +319,34 @@ export default class ProfileCommand implements BaseCommand {
...
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .MENTIONABLE,
+                    required: false,
+                    channel_types: undefined as never,
+                },
+                {
+                    name: "user_id",
+                    description:
+                        "The user ID of the player whose profile to view",
```
Comment (Brainicism) [inline src/commands/game_commands/profile.ts:342]: Same here

File: src/commands/game_commands/profile.ts [hunk 4]
```diff
@@ -399,78 +428,103 @@ export default class ProfileCommand implements BaseCommand {
...
-        return;
+    /**
+     * @param interaction - The interaction
+     * @param messageContext - The message context
+     */
+    static async processChatInputInteraction(
+        interaction: Eris.CommandInteraction,
+        messageContext: MessageContext
+    ): Promise<void> {
+        if (
```
Comment (Brainicism) [inline src/commands/game_commands/profile.ts:440]: No need for this check

File: src/commands/game_options/groups.ts [hunk 5]
```diff
@@ -325,24 +325,22 @@ export default class GroupsCommand implements BaseCommand {
...
-                } else {
-                    groups = this.getMatchedArtists(interaction.data.options);
-                }
-
-                await GroupsCommand.updateOption(
-                    messageContext,
-                    groups,
-                    interaction
-                );
+        if (
```
Comment (Brainicism) [inline src/commands/game_options/groups.ts:328]: No need for this check

**Review by Brainicism — CHANGES_REQUESTED** [review]

**Review by hoehoetvhss — APPROVED** [review]

**Review by Brainicism — DISMISSED** [review]

**Review by hoehoetvhss — APPROVED** [review]

**Pull request merged**